spec/features/orders/user_can_view_single_order_page_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'rails_helper'
+
+RSpec.feature "user can view single order" do
+  scenario "from orders page" do
+    user = create(:user)
+    visit root_path
+    click_on("Login")
+    fill_in "user[name]", with: user.name
+    fill_in "user[password]", with: "Password"
+    click_on("Log in")
+    
+    order = create(:order, user: user)
+    item = create(:item)
+    order_item = create(:order_item, order: order, item: item)
```

Comment:
We shouldn't be creating joins entries manually. Using the relationship does that automatically.

Reply:
This was Liam's test. But can you walk me through the factory stuff with regards to joins tables? They're still giving me problems on other projects